### DISCUSSION TITLE
Wann kommt die weekly-2024.49.1?

### DISCUSSION THREAD

**Discussion by JPRuehmann** [tine-groupware/tine#91] [General]
Wenn mann versucht die weekly-2024.49.1 herunterzuladen erhält man nur eine 27 Byte große Datei mit Fehler 404 als Inhalt.

**Comment by pschuele** [comment]
weekly 50 sieht wieder gut aus: https://github.com/tine-groupware/tine/releases/tag/weekly-2024.50.1
